doc/tutorials/content/sources/iccv2011/src/openni_capture.cpp

```diff
@@ -7,8 +7,8 @@
 
 OpenNICapture::OpenNICapture (const std::string& device_id)
   : grabber_ (device_id)
-  , most_recent_frame_ ()
```

Comment:
This seems odd, does the order matter here?

Reply:
That is just to make the `-Wreorder` warning go away, which demands that all fields are initialized in the order in which they are declared in the header file. Here it does not matter, but in [some cases it does](https://stackoverflow.com/questions/1828037/whats-the-point-of-g-wreorder)